lib/manager/gradle-wrapper/update.ts

```diff
@@ -35,11 +35,11 @@ export async function updateDependency(
   }
 }
 
-function replaceType(url: string) {
+function replaceType(url: string): string {
   return url.replace('bin', 'all');
 }
 
-async function getChecksum(url: string) {
+async function getChecksum(url: string): Promise<any> {
```

Comment:
We should add a type here.

Reply:
Okay, let's assume `response.body` as `string` here.